WordPressAuthenticator/NUX/NUXButton.swift

```diff
@@ -1,54 +1,285 @@
 import UIKit
 import WordPressShared
+import WordPressUI
+
+/// A protocol for an element that can display a UIActivityIndicatorView
+@objc public protocol ActivityIndicatorButton {
+    func showActivityIndicator(_ show: Bool)
+}
 
 /// A stylized button used by Login controllers. It also can display a `UIActivityIndicatorView`.
-@objc open class NUXButton: NUXSubmitButton {
-    // MARK: - Configuration
-    fileprivate let horizontalInset: CGFloat = 20
```

Comment:
Where'd these config values go?

Reply:
They moved to here: https://github.com/wordpress-mobile/WordPressUI-iOS/blob/develop/WordPressUI/Extensions/UIImage%2BAssets.swift#L87

The NUXButton was using the same sizing and configurations as `FancyButton`, so I moved the definitions out of `FancyButton` and into the `UIImage+Assets` extension. This PR has more details https://github.com/wordpress-mobile/WordPressUI-iOS/pull/8